packages/testing/config/jest/api/jest-preset.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path')
+
+const {
+  getPaths,
+  getApiSideDefaultBabelConfig,
+} = require('@redwoodjs/internal')
+
+const rwjsPaths = getPaths()
+const NODE_MODULES_PATH = path.join(rwjsPaths.base, 'node_modules')
+const { babelrc } = getApiSideDefaultBabelConfig()
+
+// @TODO
+// @TODO Not sure this logic is super solid
+// @TODO
+if (process.env.SKIP_DB_PUSH !== '1') {
+  const process = require('process')
+  const path = require('path')
+  // Load dotenvs
+  require('dotenv-defaults/config')
+
+  const cacheDirDb = `file:${path.join(__dirname, '.redwood', 'test.db')}`
+  process.env.DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
+
+  const execa = require('execa')
+  execa.sync(
+    `yarn rw`,
+    ['prisma db push', '--force-reset', '--accept-data-loss'],
+    {
+      cwd: rwjsPaths.api.base,
+      stdio: 'inherit',
+      shell: true,
+      env: {
+        DATABASE_URL: process.env.DATABASE_URL,
+      },
+    }
+  )
+
+  // If its been reset once, we don't need to re-run it for every test
+  process.env.SKIP_DB_PUSH = '1'
+}
+
+module.exports = {
+  // To make sure other config option which depends on rootDir use
+  // correct path, for example, coverageDirectory
+  rootDir: rwjsPaths.base,
+  roots: [path.join(rwjsPaths.api.src)],
+  runner: path.join(__dirname, '../jest-serial-runner.js'),
+  testEnvironment: path.join(__dirname, './RedwoodApiJestEnv.js'),
+  displayName: {
+    color: 'redBright',
+    name: 'api',
+  },
+  collectCoverageFrom: [
+    '**/*.{js,jsx,ts,tsx}',
+    '!**/node_modules/**',
+    '!**/dist/**',
+  ],
+  coverageDirectory: path.join(rwjsPaths.base, 'coverage'),
```

Comment:
Any reason we specify `coverageDirectory` here in the api preset but not in the web preset?